### ISSUE TITLE
Imagemap only works if the user is connected to the UFHealth network

### ISSUE THREAD

**Issue by marlycormar** [ctsit/imagemap#37]
Imagemap doesn't work in our production server if the user is not connected to the UFHealth network. To test this failure, connect to another network (use your phone for example) and log in to REDCap. Then, open a project with an imagemap example and the module will not work. However, the same steps are successful if connected to the UFHealth network. 

Taryn and I are experiencing the same issue. She hasn't seen this failure in other modules.

**Comment by ChemiKyle** [comment]
@marlycormar I just created a project on prod using the instrument example included with the module and it worked as expected (though I had to type in a record ID instead of being presented with the green "Add New Record" button). Could you please add me to a project where this behavior is seen?

**Comment by ChemiKyle** [comment]
Now that I'm on my home network, I can see there are 5 calls to `https://redcap.ctsi.ufl.edu/redcap/api/` that are unable to be completed. These are cryptically referred to as resulting from `index.php` on line 66. For some reason the required JS files are not being loaded while not on a UF network.

**Comment by marlycormar** [comment]
@ChemiKyle thanks for fixing issue #37!

Tested in network & off network and works in both. The console doesn't show any errors. 
Keeping up with the original design, what about doing this:
```
global $auth_meth;
$ext_path = $auth_meth == 'shibboleth' ? $this->getUrl($path, true, true) : $ext_path = $this->getUrl($path, true, false);
```

Will leave it to your discretion.

**Comment by ChemiKyle** [comment]
@marlycormar, speaking with @pbchase on Friday he was under the impression that using the API endpoint was outdated; I originally had an implementation similar to your proposal but the current one was his idea.

**Comment by marlycormar** [comment]
Fixed by PR #38.
